Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/examples.txt

```diff
@@ -18,6 +18,8 @@ cs-falcon-list-quarantined-file limit=1
 cs-falcon-apply-quarantine-file-action filename=test.exe action=delete comment=demo-comment
 cs-falcon-ods-query-scan
 cs-falcon-ods-query-scheduled-scan
+cs-falcon-ods-create-scan host_groups=7471ba0636b34cbb8c65fae7979a6a9b scan_inclusions=* cpu_priority=Highest max_duration=1 pause_duration=1
+cs-falcon-ods-create-scheduled-scan host_groups=7471ba0636b34cbb8c65fae7979a6a9b schedule_interval=daily schedule_start_timestamp=tomorrow cpu_priority=Highest scan_inclusions=*
 cs-falcon-ods-query-scan-host
 cs-falcon-ods-query-malicious-files
```

Comment:
these commands dont have args?

Reply:
they have but dont need (it doesnt change things much) let me know if i should add